app/src/store/StoreAlerts.re

```diff
@@ -0,0 +1,30 @@
+open Molecule.Alert;
+
+type alert = {
+  severity,
+  message: string,
+  id: int,
+};
+
+type action =
+  | Show(alert)
+  | Remove(alert);
+
+type state = {
```

Comment:
If we get rid of the id handling here, see comment below, we can have state be as simple as: `type state = array(alert)`.